src/acquisition/covid_hosp/facility/database.py

```diff
@@ -7,213 +7,467 @@
+            int,
+        ),
+        Columndef(
+            "total_pediatric_patients_hospitalized_confirmed_and_suspected_covid_7_day_avg",
+            "total_pediatric_patients_hosp_confirmed_suspected_covid_7d_avg",
+            float,
+        ),
+        Columndef(
+            "total_pediatric_patients_hospitalized_confirmed_and_suspected_covid_7_day_coverage",
+            "total_pediatric_patients_hosp_confirmed_suspected_covid_7d_cov",
+            int,
+        ),
+        Columndef(
+            "total_pediatric_patients_hospitalized_confirmed_and_suspected_covid_7_day_sum",
+            "total_pediatric_patients_hosp_confirmed_suspected_covid_7d_sum",
+            int,
+        ),
+        Columndef(
+            "total_pediatric_patients_hospitalized_confirmed_covid_7_day_avg",
+            "total_pediatric_patients_hospitalized_confirmed_covid_7_day_avg",
+            float,
+        ),
+        Columndef(
+            "total_pediatric_patients_hospitalized_confirmed_covid_7_day_coverage",
+            "total_pediatric_patients_hosp_confirmed_covid_7d_cov",
+            int,
+        ),
+        Columndef(
+            "total_pediatric_patients_hospitalized_confirmed_covid_7_day_sum",
+            "total_pediatric_patients_hospitalized_confirmed_covid_7_day_sum",
+            int,
+        ),
+        Columndef(
+            "total_personnel_covid_vaccinated_doses_all_7_day",
+            "total_personnel_covid_vaccinated_doses_all_7_day",
+            int,
+        ),
+        Columndef(
+            "total_personnel_covid_vaccinated_doses_all_7_day_sum",
+            "total_personnel_covid_vaccinated_doses_all_7_day_sum",
+            int,
+        ),
+        Columndef(
+            "total_personnel_covid_vaccinated_doses_none_7_day",
+            "total_personnel_covid_vaccinated_doses_none_7_day",
+            int,
+        ),
+        Columndef(
+            "total_personnel_covid_vaccinated_doses_none_7_day_sum",
+            "total_personnel_covid_vaccinated_doses_none_7_day_sum",
+            int,
+        ),
+        Columndef(
+            "total_personnel_covid_vaccinated_doses_one_7_day",
+            "total_personnel_covid_vaccinated_doses_one_7_day",
+            int,
+        ),
+        Columndef(
+            "total_personnel_covid_vaccinated_doses_one_7_day_sum",
+            "total_personnel_covid_vaccinated_doses_one_7_day_sum",
+            int,
+        ),
+        Columndef(
+            "total_staffed_adult_icu_beds_7_day_avg",
+            "total_staffed_adult_icu_beds_7_day_avg",
+            float,
+        ),
+        Columndef(
+            "total_staffed_adult_icu_beds_7_day_coverage",
+            "total_staffed_adult_icu_beds_7_day_coverage",
+            int,
+        ),
+        Columndef(
+            "total_staffed_adult_icu_beds_7_day_sum", "total_staffed_adult_icu_beds_7_day_sum", int
+        ),
+        Columndef("zip", "zip", str),
```

Comment:
The code wasn't consistent before, but even if it was, black would do this sort of inconsistent thing. Probably my least favorite part about it.

The thing black recommends here is to format these lines manually, then surround the whole block with fmt: off and fmt: on, so black doesn't touch it in the future.